en/controllers/middleware.rst

```diff
@@ -446,6 +447,42 @@ an option. You can also define your own parsers::
         return Csv::parse($body);
     });
 
+.. _https-enforcer-middleware:
+
+HTTPs Enforcer Middleware
+=========================
+
+If you want your application to only be available via HTTPS connections you can
+use the ``HttpsEnforcerMiddleware``::
+
+    use Cake\Http\Middleware\HttpsEnforcerMiddleware;
+
+    // GET requests will be automatically redirect to HTTPS if they
+    // are created with HTTP.
+    $https = new HttpsMiddleware([
+        'redirect' => true,
```

Comment:
`redirect` defaults to `true`. 

Examples with non-default `redirect/statusCode/disableOnDebug` could be better.